src/olympia/templates/copyright.html

```diff
@@ -1,13 +1,13 @@
 <div id="copyright">
   <p id="footer-links">
     {% block footer_links %}
-      <a href="http://www.mozilla.org/privacy/websites/">
+      <a href="https://www.mozilla.org/en-US/privacy/websites/">
         {{ _('Privacy Policy') }}
       </a> &nbsp;|&nbsp;
-      <a href="http://www.mozilla.org/about/legal.html">
+      <a href="https://www.mozilla.org/en-US/about/legal/">
         {{ _('Legal Notices') }}
       </a> &nbsp;|&nbsp;
-      <a href="http://www.mozilla.org/legal/fraud-report/index.html">
+      <a href="https://www.mozilla.org/en-US/about/legal/fraud-report/">
```

Comment:
Give the other 2 links are localized is this link likely to be updated to be localized too? If it is it would be better to leave off the locale as @wagnerand originally suggested.